.github/path-ci/run-smoke.ps1

```
@@ -0,0 +1,93 @@
+Write-Host '== server =='
+$out = Invoke-FountCapture server
+Assert-OutputContains server $Marker $out
+
+Write-Host '== reboot =='
+$out = Invoke-FountCapture reboot
+Assert-OutputContains reboot $Marker $out
+Assert-OutputContains reboot reboot $out
+
+Write-Host '== log =='
+$out = Invoke-FountCapture log
+Assert-OutputContains log 'FOUNT_CI_HOOK:log' $out
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Assert-OutputContains' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)